conanfile.py

```diff
@@ -0,0 +1,52 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class QtProtobufConan(ConanFile):
+    name = "qtprotobuf"
+    version = "0.4.0"
+    license = "MIT"
+    url = "https://github.com/semlanik/qtprotobuf"
+    description = ("gRPC and Protobuf generator and bindings for Qt framework")
+    topics = ("conan", "qt", "protobuf")
+    settings = "os", "compiler", "build_type", "arch"
+    homepage = "https://github.com/semlanik/qtprotobuf"
+    options = {"shared": [True, False]}
```

Comment:
Have feeling that this option should control build type(static/shared library), but not used. Also probably it's better to build shared library by default.

Reply:
`shared` is a special option. Conan CMake helper sets BUILD_SHARED_LIBS if `shared` is True.

Reply:
Ok, I had some concerns about this. Let's keep it as is, but I switched BUILD_SHARED_LIBS option off forcibly in CMake because of hacked static build. Will rework whole static build chain later. Thanks :)